mono/mini/mini-generic-sharing.c

```diff
@@ -763,6 +767,9 @@ free_inflated_info (MonoRgctxInfoType info_type, gpointer info)
 	case MONO_RGCTX_INFO_CAST_CACHE:
 		mono_metadata_free_type ((MonoType *)info);
 		break;
+	case MONO_RGCTX_INFO_DELEGATE_TRAMP_INFO:
+		g_free (info);
```

Comment:
Is this alway the right free?  Can this pointer come from the `mono_image_alloc0` aka `temporary == false` case?

Reply:
The convention is that this function `free_inflated_info` is only called for `temporary == true` cases. So this should be correct if there is not a bug in any calling code.